types/index.ts

```diff
@@ -183,3 +160,9 @@ declare global {
     ): void;
   }
 }
+
+// Top level API interfaces
+
+export interface Edkt {
+  run: () => Promise<void>;
+}
```

Comment:
Should we describe what this takes?

Reply:
And actually use it, yes, definitely!
Let me fix that, RQ!